### DISCUSSION TITLE
posts/Go/zap-otel/

### DISCUSSION THREAD

**Discussion by giscus** [Q1mi/BlogComments#371] [Announcements]
# posts/Go/zap-otel/

李文周的 Blog 提供 Go 语言中文教程,李文周博客,Go 语言学习之路是我整理的一个免费教程，包含部分视频教程，也是非常适合新手学习Go语言的入门教程。本文介绍了 Go 语言 zap日志库如何配置链路追踪，以及如何在日志中记录traceID。 

https://www.liwenzhou.com/posts/Go/zap-otel/

**Comment by xiaozhaofu** [comment]
没有WithTraceIDField这个函数呢，源码里也没有和TraceId相关的呢
